uisdk/src/main/java/com/karhoo/uisdk/screen/booking/checkout/payment/braintree/BraintreePaymentPresenter.kt

```diff
@@ -166,7 +166,7 @@ class BraintreePaymentPresenter(view: PaymentDropInContract.Actions,
         return currency.intToPriceNoSymbol(quote?.price?.highPrice.orZero())
     }
 
-    override fun sdkInit(quote: Quote?) {
+    override fun sdkInit(quote: Quote?, context: Context?) {
```

Comment:
Why dont we inject the Locale instead of Context? Would it be better to inject Locale to avoid memory leaks?